src/sim-host/ui/custom-elements.js

```diff
@@ -412,6 +480,14 @@ function registerCustomElement(name) {
     });
 }
 
+function getId() {
+    var result = [];
+    for (var i = 0; i < 8; i++) {
+        result.push((Math.random() * 16 | 0).toString(16));
```

Comment:
If the idea is to ensure everything gets a unique ID, why not just use an incrementing counter?